util/src/main/java/io/zeebe/util/sched/ActorThread.java

```diff
@@ -173,7 +173,7 @@ public int getRunnerId() {
   }
 
   @Override
-  public void start() {
+  public synchronized void start() {
```

Comment:
Can you elaborate why the `synchronized`?

Reply:
https://lgtm.com/rules/5990064/